scipy/optimize/_minimize.py

```diff
@@ -65,6 +65,11 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
         where ``x`` is a 1-D array with shape (n,) and ``args``
         is a tuple of the fixed parameters needed to completely
         specify the function.
+
+        Suppose the callable has signature ``f0(x, *args, **kwargs)``, where ``args``
+        and ``kwargs`` are required positional and keyword arguments. Rather than
+        passing ``f0`` as the callable, consider wrapping it to accept only ``x``;
+        e.g., pass ``fun=lambda x: f0(x, *args, **kwargs)`` as the callable.
```

Comment:
```suggestion
        Suppose the callable has signature ``f0(x, *args, **kwargs)``, where ``args``
        and ``kwargs`` are required positional and keyword arguments
        (not to be confused with the `args` argument to this function, which we
        suggest you disregard). Rather than
        passing ``f0`` as the callable, consider wrapping it to accept only ``x``;
        e.g., pass ``fun=lambda x: f0(x, *my_args, **my_kwargs)`` as the callable,
        where `args` (tuple) and `kwargs` (dict) have been gathered before 
        invoking this function.
```

Reply:
So I get renaming `my_args` to not conflict with `args` - how about changing it throughout? It should be the same in lines 74 and 75, at least, but why not in 69?

Reply:
You're right; early morning!

Reply:
OK, so when we change `args` and `kwargs` to `my_args` and `my_kwargs`, this could become:

```suggestion
        Suppose the callable has signature ``f0(x, *my_args, **my_kwargs)``, where 
        ``my_args`` and ``my_kwargs`` are required positional and keyword arguments.
        Rather than passing ``f0`` as the callable, consider wrapping it to accept
        only ``x``; e.g., pass ``fun=lambda x: f0(x, *my_args, **my_kwargs)`` as the
        callable, where `my_args` (tuple) and `my_kwargs` (dict) have been gathered 
        before invoking this function.
```
Here, I've removed "not to be confused with the `args` argument to this function, which we suggest you disregard)" because `my_args` and `args` have different names, so it is difficult to confuse the two.

Reply:
I am also fine with this last revision